actionpack/lib/action_dispatch/log_subscriber.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module ActionDispatch
+  class LogSubscriber < ActiveSupport::LogSubscriber
+    def redirect(event)
+      payload = event.payload
+      status = payload[:status]
+
+      message = +"Completed #{status} #{Rack::Utils::HTTP_STATUS_CODES[status]} in #{event.duration.round}ms"
+      message << "\n\n" if defined?(Rails.env) && Rails.env.development?
```

Comment:
I thought we try to avoid calling `env` in Rails code and try to use configuration in Railties instead. For example: https://github.com/rails/rails/blob/2f53d6c31e71732f342aaa9542b771261baa6866/railties/lib/rails/application/configuration.rb#L36

But as we do the same for `ActionController::LogSubscriber` I'm not sure 🤷

Reply:
Yeah this code is basically a copy of what happens in `ActionController::LogSubscriber`, so not sure either :sweat_smile:

https://github.com/rails/rails/blob/a797083d72e21484ba7ffe89743ea7ccb6c62344/actionpack/lib/action_controller/log_subscriber.rb#L34